packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -189,21 +189,28 @@ const runGenerator = (answers, {outputDir}) => {
         p.resolve(outputDir, 'app', 'api.config.js')
     )
 
-    npmInstall(outputDir)
+    npmInstall(outputDir, {verbose})
 }
 
-const npmInstall = (outputDir) => {
+const npmInstall = (outputDir, {verbose}) => {
     console.log('Installing dependencies for the generated project. This may take a few minutes.\n')
-    child_proc.execSync('npm install --quiet', {
-        cwd: outputDir,
-        stdio: 'inherit',
-        env: {
-            ...process.env,
-            OPENCOLLECTIVE_HIDE: 'true',
-            DISABLE_OPENCOLLECTIVE: 'true',
-            OPEN_SOURCE_CONTRIBUTOR: 'true'
-        }
-    })
+    const npmLogLevel = verbose ? 'notice' : 'error'
+    try {
+        child_proc.execSync(`npm install --color always --loglevel ${npmLogLevel}`, {
+            cwd: outputDir,
+            stdio: verbose ? 'inherit' : ['inherit', 'ignore', 'inherit'],
```

Comment:
Just glancing at this and wanting to note that `['inherit', 'ignore', 'inherit']` in this line corresponds to `stdin`, `stdout`, `stderr` in that order. So we disable stdout but leave the other untouched.

This is also documented in more detail here: https://nodejs.org/api/child_process.html

Reply:
@kevinxh as Vincent mentioned above, I'm thinking for better readability, maybe have something like this:

```suggestion
            stdio: verbose ? 'inherit' : disableStdOut,
```